### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Shaheryar0054**
### Hello Microverse milestone 1: Models
In this milestone we:

- Added the git ignore file
- Added linters for automation
- Added HTML and CSS file
- Added test.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,55 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (franclobo) [inline README.md]: - [ ] Kindly add "Microverse" in the acknowledgments. 👆🏽

**Review by roshan-bajgain — COMMENTED** [review]
Hi @Shaheryar0054, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a different pull request. you have submitted the `hello-microverse` project on ` Portfolio: setup and mobile first`. Please place it into right place 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/test.md
```
Comment (franclobo) [inline .gitignore:2]: - [ ] Please 🙏🏽 add `node_modules/` on the `.gitignore` file.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Shaheryar0054, 

Good job so far!
You are doing it well 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive commits 💯 
- Professional README file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please 🙏🏽 add a descriptive pull request complete your PR with a sumary of activities.
image

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Shaheryar0054) [reply README.md]: okay

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @Shaheryar0054  👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## [Optional] suggestions:

[ ] kindly remove the ` / ` from the `.gitignore` so your `test.md ` gets ignored by git .

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**